packages/pipeline/src/parsers/token_metadata/index.ts

```diff
@@ -0,0 +1,47 @@
+import * as R from 'ramda';
+
+import { MetamaskTrustedTokenMeta, ZeroExTrustedTokenMeta } from '../../data_sources/trusted_tokens';
+import { TokenMetadata } from '../../entities';
+
+/**
+ * Parses Metamask's trusted tokens list.
+ * @param rawResp raw response from the metamask json file.
```

Comment:
nit: The raw response comes from an HTTP request, not a JSON file.

Reply:
k i'll edit in-line